src/axom/inlet/Inlet.hpp

```diff
@@ -275,16 +277,128 @@ class Inlet : public SchemaCreator
    */
   void writeDoc();
 
+  /*!
+   *****************************************************************************
+   * \brief Verifies the contents of the sidreGroup according to Inlet 
+   * requirements.
+   *
+   * This checks the correctness of each Field and Table in the input deck.
+   * 
+   * \return true if contents are correct and false if not.
+   *
+   *****************************************************************************
+   */
+  bool verify(); 
+
   // TODO add update value functions
 private:
+  /*!
+   *****************************************************************************
+   * \brief Verifies the contents of the sidreGroup according to Inlet 
+   * requirements.
+   *
+   * This recrusively checks the correctness of each Field and Table in the sidre
```

Comment:
Typo in recursively...

I would move most of this description to the verify() function since that is what the users will read.  This description can just be something simple like "recursive internal implementation of verify()"